### PR TITLE
chore: add PR template

### PR DESCRIPTION
**Pull request by christjt**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Please also include relevant motivation and context. 
+List any dependencies that are required for this change.
+
+
+# Checklist:
+
+Here is a checklist that should completed before merging this given feature. 
+Any shortcomings from the items below should be explained and detailed within the contents of this PR.
+
+- [ ] I am proud of of the feature.
```
Comment (haakonflatval-cognite) [inline .github/pull_request_template.md:13]:
```suggestion
- [ ] I am proud of the feature.
```
Just making you more proud of this PR :angel:

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- [ ] I have performed a self-review of my own code.
+- [ ] I have tested manually tested this for different scenarios (different models, formats, devices, browsers).
+- [ ] I have commented my code in hard-to-understand areas.
+- [ ] I have made corresponding changes to the public documentation.
+- [ ] I have added unit and visuals test that prove my that my feature works to the best of my ability.
```
Comment (Savokr) [inline .github/pull_request_template.md:18]: prove mE?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+List any dependencies that are required for this change.
+
+
+# Checklist:
+
+Here is a checklist that should completed before merging this given feature. 
+Any shortcomings from the items below should be explained and detailed within the contents of this PR.
+
+- [ ] I am proud of of the feature.
+- [ ] I have performed a self-review of my own code.
```
Comment (haakonflatval-cognite) [inline .github/pull_request_template.md:14]: This is quite an extensive list. Is every bullet point intended to be mandatory for all (feature) PRs?

Comment (christjt) [reply .github/pull_request_template.md:14]: Yup. I am open to suggestions though. What should not be included?

**Review by pramodcog — COMMENTED** [review]
Can we add Jira update with status, findings & other relevant information's into this checklist

**Pull request merged**